susemanager-utils/susemanager-sls/salt/certs/OEL6.sls

```diff
@@ -0,0 +1 @@
+RedHat6.sls
```

Comment:
For reference, I failed originally because I created `Oracle*.sls` files.

Here are the grains from an Oracle Linux 8:
```
    os:
        OEL
    os_family:
        RedHat
    osarch:
        x86_64
    oscodename:
        Oracle Linux Server 8.2
    osfinger:
        Oracle Linux Server-8
    osfullname:
        Oracle Linux Server
    osmajorrelease:
        8
    osrelease:
        8.2
    osrelease_info:
        - 8
        - 2